.claude/scripts/worktree-cleanup-all.sh

```
@@ -0,0 +1,86 @@
+  local path=$1 label toplevel expected
+  [ -d "$path/.claude/worktrees" ] || return 0
+  # Only sweep a repo whose toplevel resolves to ITSELF. A submodule with broken
+  # worktree isolation resolves into the main checkout, and sweeping through that
+  # alias would operate on the wrong tree (AGENTS.md, Execution model).
+  toplevel=$(git -C "$path" rev-parse --show-toplevel 2>/dev/null) || return 0
+  expected=$(cd "$path" && pwd -P) || return 0
+  if [ "$toplevel" != "$expected" ]; then
+    printf '\n### SKIP %s (broken isolation: toplevel=%s)\n' "$path" "$toplevel"
+    return 0
+  fi
+  local rel=${path#$ROOT/}
```

Shellcheck (0.11.0)

[info] Expansions inside ${..} need to be quoted separately, otherwise they match as patterns.

(SC2295)